scale-encode/Cargo.toml

```diff
@@ -17,7 +17,7 @@ include.workspace = true
 default = ["std", "derive", "primitive-types", "bits"]
 
 # Activates std feature.
-std = ["scale-info/std"]
+std = []
```

Comment:
do we need the `std` feature now?

Reply:
also scale-bits brings in scale-info, is that intended?

Reply:
We still use it to impl `Error` on the error types; such a small silly thing; we are very close to just not needing it at all (same as in a bunch of these crates; it'd be awesome if they were just 100% no-std) :(

Reply:
I should have checked the `cargo tree`; `scale-decode` also still brings in scale-info via scale-type-resolver, whoops!

Reply:
one step at the time :)